pkg/appruntime/app_runtime.go

```diff
@@ -151,6 +154,26 @@ func (a *Application) Run(ctx context.Context, cli client.Client, respStream cha
 	if a.Spec.DocNullReturn != "" {
 		out[base.APPDocNullReturn] = a.Spec.DocNullReturn
 	}
+	if input.ConversationID != "" {
+		conversationKnowledgebaseExist := true
+		kb := &arcadiav1alpha1.KnowledgeBase{}
+		err := cli.Get(ctx, types.NamespacedName{Namespace: a.Namespace, Name: input.ConversationID}, kb)
+		if err != nil {
+			if apierrors.IsNotFound(err) {
+				conversationKnowledgebaseExist = false
```

Comment:
if there is no document loaded, will this log be always logged? or the ConversationID will be only present if there is document loaded? maybe we should check if any docs in the conversation?

Reply:
When a document is uploaded in a conversation, a knowledgebase named conversation ID is created. code in #873 

The logic here is that:
1. only if there is a conversation ID (means not a new conversation)
2. and this conversation knowledgebase exists
3. and the state is ready (embedding has been completed) 
will it be introduced into the subsequent chat.

Reply:
I mean, for conversation that doesn't have any document uploaded, will there'll be some unexpected logs?

Reply:
we can change V(3) -> V(5)